spec/rails_helper.rb

```diff
@@ -0,0 +1,3 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
```

Comment:
What's the purpose of this new file? It's only used in one spec, and we could just require `spec_helper` there directly, no?

Reply:
I've removed this since I also didn't see the reason for it, but maybe @mkllnk can shed some light here once he's back.

Reply:
Actually it looks like that change breaks a bunch of tests, I put it back :)

Reply:
Hm something strange is going on here with the build, these test are all failing in CI but passing locally for me. I'm going to try to figure out what's going on here, maybe hold off on a re-review for now @Matt-Yorkley

Reply:
This file is a [new convention](https://relishapp.com/rspec/rspec-rails/v/3-9/docs/upgrade) since rspec-rails-3. The intention is to use the spec_helper for specs that don't need rails and use the rails_helper for rails-dependent specs.

My main reason to include it here was that I used `rails generate` and it referenced the rails_helper. Adding the rails_helper means that we don't need to change that line every time we use a generator. It might also feel more familiar to newcomers who just know the new rails conventions.

Our current spec_helper has over 200 lines. Splitting some of that into two different files with different purposes might be a good idea as well.